doc/getting-started/acrn-sanity-test.rst

```diff
@@ -0,0 +1,349 @@
+ACRN Sanity Test
+################
+
+The document describes the daily sanity test for ACRN by validation team, which 
+includes a set of system level core tests of basic functionality to demonstrate 
+ACRN hypervisor’s availability. User could manually run these after setup ACRN. 
+
```

Comment:
I have just realised that you use the pre-installed script (`launch_uos.sh`) in these tests. It assumes the system has a user called `clear` already set-up. We need to make this explicit in this document as most people will set-up a different user.

Reply:
Are you going to add some text here to address this comment or will @deb-intel do it?